src/t8_cmesh/t8_cmesh_vertex_connectivity/t8_cmesh_vertex_connectivity.cxx

```diff
@@ -76,18 +56,21 @@ t8_cmesh_get_num_local_vertices (const t8_cmesh_t cmesh)
 }
 
 const t8_gloidx_t *
-t8_cmesh_get_global_vertices_of_tree (const t8_cmesh_t cmesh, const t8_locidx_t local_tree, const int num_vertices)
+t8_cmesh_get_global_vertices_of_tree (const t8_cmesh_t cmesh, const t8_locidx_t local_tree, int *num_vertices)
```

Comment:
`num_vertices` should match the data type of std::span.size() which is `size_t`.
This holds for the other functions as well.

Reply:
This gets really nasty with type conversions because the global vertices themselves are `t8_gloidx_t`. We should bast change the global vertices to an unsigned datatype which makes more sense. Then we also do not need to cast everything back and forth. I will open an issue for this